src/generators/new/packagejson.js

```diff
@@ -61,22 +63,24 @@ module.exports = async ({
 
   const packages = [
     '@herbsjs/herbs2gql',
-    'graphql@15.5.1',
-    'apollo-server-express@2.25.2',
-    'apollo-server@2.25.2',
     'sugar-env@1.5.14',
     'dotenv@10.0.0',
     '@herbsjs/herbsshelf',
-    '@herbsjs/herbs2rest',
-    'graphql-tools@8.1.0',
     'nodemon',
-    'lodash.camelcase'
+    'lodash.camelcase',
+    'apollo-server-express@2.25.2'
   ]
 
   if (mongo) { packages.push('mongodb') }
   if (postgres) {
     packages.push('@herbsjs/herbs2knex')
     packages.push('pg')
   }
+  if (rest) { packages.push('@herbsjs/herbs2rest') }
```

Comment:
I guess here will break now, because herbs2rest needs express(     "express": "^4.17.1" ) to run, that was installed with graphql.

Could you please check this before merge? Thank you

Reply:
This dependency is mapped in the general listing using the apollo-server-express library. 

> using apollo-server-express we can serve both REST and GraphQL at the same time using Express [Reference](https://www.apollographql.com/blog/backend/using-express-with-graphql-server-node-js/)

Anyway, I did tests generating projects using the options and it worked as expected